multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/DetectApplicationsToRenameStep.java

```diff
@@ -49,13 +51,28 @@ protected StepPhase executeStep(ProcessContext context) {
         return StepPhase.DONE;
     }
 
-    private List<String> computeOldAppsToRename(Set<String> appsToProcess) {
-        return appsToProcess.stream()
-                            .filter(appName -> !BlueGreenApplicationNameSuffix.isSuffixContainedIn(appName))
-                            .collect(Collectors.toList());
+    private List<String> computeOldAppsToRename(DeployedMta deployedMta, List<String> selectedModulesList) {        
+        Set<String> selectedModulesSet = new TreeSet<>();
```

Comment:
I don't understand the need for this variable.
Just pass the list (and remove the `List` suffix from its name) to the other method

Reply:
the set will have better performance when later making the contains check

Reply:
will remove in service of code clarity, performance isn't really a concern here